client/src/pages/course/admin/tasks.tsx

```diff
@@ -162,6 +177,7 @@ function Page(props: CoursePageProps) {
       message.error('An error occurred.');
     } finally {
       setLoading(false);
+      await loadData();
```

Comment:
why to load data after removing spinner?

Reply:
Fixed

Reply:
not sure why you removed loading handling?
also probably you could just loaddata after mutating task.